src/Morphir/TypeScript/NamespaceMerger.elm

```diff
@@ -0,0 +1,107 @@
+module Morphir.TypeScript.NamespaceMerger exposing (mergeNamespaces)
+
+{-| This module contains the TypeScript backend that translates the Morphir IR into TypeScript.
+-}
```

Comment:
No it doesn't ... :)

Reply:
Hah! 😂

I've pushed a fix. (Though I'll need to rewrite it all over again soon, when I pull all the top-level-file code into one elm file)